numba/pycc/compiler.py

```diff
@@ -261,17 +263,32 @@ def _emit_environment_array(self, llvm_module, builder, pyapi):
                                               env_defs_init)
         return gv.gep([ZERO, ZERO])
 
+    def _emit_envgvs_array(self, llvm_module, builder, pyapi):
```

Comment:
A quick comment about what this does and why would be useful for future developers looking at this function.

Reply:
Added docstr to the function